group-actors/src/main/java/org/sunbird/service/UserServiceImpl.java

```diff
@@ -105,9 +110,11 @@ public Response getOrganisationDetails(String orgId) throws BaseException {
     Response responseObj = new Response();
     SearchRequest readRequest = new SearchRequest();
     readRequest.getRequest().put(JsonKey.ORGANISATION_ID, orgId);
-
     Map<String, String> requestHeader = new HashMap<>();
-    getUpdatedRequestHeader(requestHeader);
+    Map<String, Object> requestContext = new HashMap<>();
+    requestContext.put(JsonKey.X_REQUEST_ID, UUID.randomUUID().toString());
```

Comment:
context should be fro request, not new context

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId